backend/src/main/java/ca/bc/gov/restapi/results/oracle/service/OrgUnitService.java

```diff
@@ -16,23 +18,27 @@ public class OrgUnitService {
 
   private final OrgUnitRepository orgUnitRepository;
 
+  @Value("${nr.results.config.opening-search.org-units}")
+  private String[] orgUnitsFromProps;
+
   /**
-   * Find all Org Units. Option to include expired ones.
+   * Find all Org Units for the Openings Search.
    *
-   * @param includeExpired True to include expired, false otherwise.
    * @return List of {@link OrgUnitEntity} with found categories.
    */
-  public List<OrgUnitEntity> findAllOrgUnits(boolean includeExpired) {
-    log.info("Getting all org units. Include expired: {}", includeExpired);
+  public List<OrgUnitEntity> findAllOrgUnits() {
+    log.info("Getting all org units for the search openings");
 
-    if (includeExpired) {
-      List<OrgUnitEntity> orgUnits = orgUnitRepository.findAll();
-      log.info("Found {} org units (including expired)", orgUnits.size());
-      return orgUnits;
+    if (Objects.isNull(orgUnitsFromProps) || orgUnitsFromProps.length == 0) {
+      log.info("No Org Units from the properties file.");
+      return List.of();
     }
 
-    List<OrgUnitEntity> orgUnits = orgUnitRepository.findAllByExpiryDateAfter(LocalDate.now());
-    log.info("Found {} org units (excluding expired)", orgUnits.size());
+    List<String> orgUnitCodes = Arrays.asList(orgUnitsFromProps);
+
+    List<OrgUnitEntity> orgUnits = orgUnitRepository.findAllByOrgUnitCodeIn(orgUnitCodes);
```

Comment:
You can simplify this by passing the array straight as a parameter of the repository method. There's no need to keep a variable that will not be reused later on

Reply:
Done here https://github.com/bcgov/nr-silva/pull/393/commits/c0378d713ce46d3def5152060bb31e88b8b472d7